### ISSUE TITLE
Only working once and without debugger

### ISSUE THREAD

**Issue by SirFlip** [JesusFreke/fusionIdea#10]
A similar Problem as described in #8 is occuring when I am using it.
I can only attach to Fusion (get console output) once. Afterwards the Script runs but you don't get any consol output, until I restart Fusion.
Debuging doesn't work at all it just irgnores breakpoints but executes the script.

Plugin: 0.5.7
PyCharm: PyCharm 2019.3.4 (Professional Edition) Build #PY-193.6911.25, built on March 18, 2020
Fusion 360: 2.0.7830

Sidenote: I have Anaconda installed. Tryed it with anaconda interpreter and also tried it with the Fusion one too.
Sidenote2: Debugging with VScode dosn't work for me neither. But I think thats because of VScode being VScode.

**Comment by JesusFreke** [comment]
Yeah, this is a known issue. The current versions of IDEA/PyCharm have a broken version of the python debugging infrastructure, unfortunately.

There's some work in progress to fix this, you can follow along on issue #7.

**Comment by JesusFreke** [comment]
I'm hoping to finish it up and release a new version soon. But in the meantime, if you want to try it out, you can check out and build the idea-2019.3.3 branch. The command to build the plugin is `./gradlew.bat buildPlugin`

You'll also need to install the [fusion_idea_addin](https://github.com/JesusFreke/fusion_idea_addin) in fusion 360.

**Comment by SirFlip** [comment]
Ohh i thought it was wonly about Mac so I didn't look deep into it.

I'll give the workaroud it a try.

**Comment by JesusFreke** [comment]
Yeah, it's kinda 2 different issues, but the Mac problem requires some pretty substantial changes -- a part of which will also fix these attach issues on Windows.

**Comment by SirFlip** [comment]
Ok I built it no problem there.

But I think python cannot get the files out of the jar. 
In fact the scripts aren't even in the jar.

```
C:\Users\hanne\AppData\Local\Autodesk\webdeploy\production\5d5b542f5ddb82dac8b734aa2cb375dc33c4efb2\Python\python.exe C:\Users\hanne\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\193.6911.25.plugins\fusion_idea-0.5.8.jar\scripts\initiate_attach.py --script D:\Python\Fusion360\progress.py --pid 32716 --debug 1 --pydevd_path C:\Users\hanne\AppData\Local\JetBrains\Toolbox\apps\PyCharm-P\ch-0\193.6911.25.plugins\fusion_idea-0.5.8.jar\scripts\pydevd-1.9.0 --port 63296
C:\Users\hanne\AppData\Local\Autodesk\webdeploy\production\5d5b542f5ddb82dac8b734aa2cb375dc33c4efb2\Python\python.exe: can't find '__main__' module in 'C:\\Users\\hanne\\AppData\\Local\\JetBrains\\Toolbox\\apps\\PyCharm-P\\ch-0\\193.6911.25.plugins\\fusion_idea-0.5.8.jar\\scripts\\initiate_attach.py'
```
And I must have broken something else trying to work it out because I get "Nothing to run on" statements and cant even start it anymore.

**Comment by JesusFreke** [comment]
Hmm. I think maybe you installed the wrong file. The plugin typically isn't distributed as a jar file. The installable zip for the plugin should be something like `build/distributions/fusionIdea<version>.zip`

**Comment by SirFlip** [comment]
Yeah, my bad. Out of habit from Java I used the jar from the `build/libs` folder and overlooked the `build/distributions` folder.
With the zip everything it works fine.

**Comment by JesusFreke** [comment]
Great! Thanks for reporting back. I'm hoping to get a new release out this weekend-ish, but we'll see.

**Comment by JesusFreke** [comment]
This should be fixed in v0.6.0. 

The new version of the plugin (v0.6.0) has been submitted to IDEA's plugin repository, but it may take a few days to be published. In the meantime, you can download the plugin [here ](https://github.com/JesusFreke/fusionIdea/releases/tag/v0.6.0) for manual installation.
